server/src/main/java/com/thoughtworks/go/server/service/GoConfigService.java

```diff
@@ -248,12 +248,12 @@ public boolean hasPipelineNamed(final CaseInsensitiveString pipelineName) {
         return getCurrentConfig().hasPipelineNamed(pipelineName);
     }
 
-    public PipelineConfig pipelineConfigNamed(final CaseInsensitiveString name) {
-        return getCurrentConfig().pipelineConfigByName(name);
+    public PipelineConfig editablePipelineConfigNamed(final String name) {
+        return getMergedConfigForEditing().pipelineConfigByName(new CaseInsensitiveString(name));
```

Comment:
@marques-work I am little paranoid changing this method as it is used in multiple places and as of now I don't know the repercussion of changing it.

I understand why you had to make this change as the `getCurrentConfig()` returns a preprocessed config which would have a pipeline using templates to show all the stage details.

 How bout having a new method for fetching the pipeline from `GoConfigService#getConfigForEditing()`?

Reply:
@marques-work sorry my bad, I think you have changed only the method used by `ExportControllerV1` ignore my previous comment on being used in multiple places.